lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -394,6 +403,7 @@ def add_safe_nav_to_all_methods_in_chain(corrector,
           start_method.each_ancestor do |ancestor|
             break unless %i[send block].include?(ancestor.type)
             next if !ancestor.send_type? || ancestor.operator_method?
+            next if ancestor.safe_navigation?
```

Comment:
Is this new condition necessary? If it is, please add a test that uses it.

```suggestion
```

Reply:
You're right, part of debugging and wasn't needed.